utils/docker/publish_docker.sh

```diff
@@ -113,6 +113,5 @@ publish_docker_manifest () {
   done
 }
 
-publish_docker_images_with_arch_suffix focal amd64
 publish_docker_images_with_arch_suffix focal arm64
```

Comment:
shall we drop this as well?

Reply:
had a typo, arm should have been deleted.